ci/before_deploy.sh

```diff
@@ -18,7 +18,7 @@ main() {
     test -f Cargo.lock || cargo generate-lockfile
 
     # TODO Update this to build the artifacts that matter to you
-    cross rustc --bin hello --target $TARGET --release -- -C lto
+    cross rustc --bin hello --target $TARGET --release
```

Comment:
The correct fix to avoid losing the LTO optimizations would be this:

```suggestion
    export CARGO_PROFILE_RELEASE_LTO=true
    cross build --bin hello --target $TARGET --release
```